src/test/java/hudson/plugins/parameterizedtrigger/test/FileBuildParameterFactoryTest.java

```diff
@@ -245,22 +257,31 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
             }
         });
 
-        // add Trigger builder, with file parameter factory
-        projectA.getBuildersList().add(createTriggerBuilder(projectB, NoFilesFoundEnum.SKIP, "UTF-8"));
+        // SECURITY-170: need to allow multibyte params that can't be traditionally declared.
```

Comment:
When I tested locally, I couldn't declare the multibyte params causing this test to fail.